catalog/app/utils/AWS/S3.js

```diff
@@ -124,7 +125,15 @@ function useSmartS3() {
           params = null
         }
 
+        let forceProxy = false
+        if (params && 'forceProxy' in params) {
+          forceProxy = params.forceProxy
+          delete params.forceProxy
+        }
         const req = super.makeRequest(operation, params)
+        if (forceProxy) {
+          req.httpRequest[FORCE_PROXY] = true
+        }
```

Comment:
I think you can always try to `delete params.forceProxy` and replace forceProxy as boolean flag:

```ts
const isForceProxy = params && 'forceProxy' in params
delete params.forceProxy
const req = super.makeRequest(operation, params)
if (isForceProxy) {
  req.httpRequest[FORCE_PROXY] = true
}
```

This way you don't use `let` and reassign variable, that I intuitively avoid